src/OrbitGl/LineGraphTrack.cpp

```diff
@@ -24,7 +24,7 @@ namespace orbit_gl {
 
 template <size_t Dimension>
 [[nodiscard]] static std::array<float, Dimension> GetNormalizedValues(
-    const std::array<double, Dimension>& values, double min, double inverse_value_range) {
+    absl::Span<const double> values, double min, double inverse_value_range) {
```

Comment:
Nit: All the changes here in LineGraphTrack seem to be unrelated for the current PR. Would you mind moving them into the next PR (or whenever you remove Dimension from LineGraphTrack)?

Reply:
Changed accordingly. Thanks for the suggestion!

This was for the purpose of avoiding the vector to array conversion in `LineGraphTrack`.